db/seed.js

```diff
@@ -0,0 +1,54 @@
+const faker = require('faker');
+const Location = require('./db.js')
+
+for (var i = 1; i <= 10; i++) {
+  const samplePosts = 
+    { locationID: i,
+      reviews: [],
+    }
+  ;

```

Comment:
@harrisonywu @krise3 semi-colon should be immediately after the curly brace. The object on lines 6-8 should probably just be on line 5.